spec/ddtrace/benchmark/pg_contrib_spec.rb

```diff
@@ -0,0 +1,263 @@
+# typed: ignore
```

Comment:
Thank you for adding this benchmark!
Could you capture a sample result from it and add it to this PR's description?